src/views/CountryRiskWatch/RiskTable/index.tsx

```diff
@@ -0,0 +1,257 @@
+                (option) => riskScoreToCategory(
+                    getDisplacementForSelectedMonths(
+                        selectedMonths,
+                        informSeasonalRiskData[option.hazard_type],
+                        'max',
+                    ),
+                    option.hazard_type,
+                ),
+                {
+                    headerInfoTitle: strings.riskTableInformTitle,
+                    headerInfoDescription: (
+                        <div className={styles.informDescription}>
+                            <div>
+                                {strings.riskTableInformDescriptionP1}
+                            </div>
+                            <div>
+                                {strings.riskTableInformDescriptionP2}
+                            </div>
+                            <div>
+                                {resolveToComponent(
+                                    strings.riskTableInformDescriptionP3,
+                                    {
+                                        moreInfoLink: (
+                                            <Link to="https://drmkc.jrc.ec.europa.eu/inform-index/INFORM-Covid-19/INFORM-Covid-19-Warning-beta-version">
+                                                {strings.riskTableInformDescriptionHereLabel}
+                                            </Link>
+                                        ),
+                                    },
+                                )}
+                            </div>
+                        </div>
+                    ),
+                },
+            ),
+            createNumberColumn<HazardTypeOption, string>(
+                'exposure',
+                strings.riskTablePeopleExposedTitle,
+                (option) => getDisplacementForSelectedMonths(
+                    selectedMonths,
+                    exposureRiskData[option.hazard_type],
+                ),
+                {
+                    headerInfoTitle: strings.riskTablePeopleExposedTitle,
+                    headerInfoDescription: strings.riskTablePeopleExposedDescription,
+                    maximumFractionDigits: 0,
+                },
+            ),
+            createNumberColumn<HazardTypeOption, string>(
+                'displacement',
+                strings.riskTableDisplacementTitle,
+                (option) => {
+                    const exposure = getDisplacementForSelectedMonths(
+                        selectedMonths,
+                        exposureRiskData[option.hazard_type],
+                    );
+
+                    const displacement = getDisplacementForSelectedMonths(
+                        selectedMonths,
+                        displacementRiskData[option.hazard_type],
+                    );
+
+                    if (isNotDefined(displacement)) {
+                        return undefined;
+                    }
+
+                    return minSafe([exposure, displacement]);
```

Comment:
Why do we return undefined when displacement is undefined but not the same behavior for exposure? Can we add a NOTE?

Reply:
Because, it's the displacement column, and we don't want to show the exposure if the displacement is undefined, but we want to show the exposure in the case that displacement > exposure (which cannot occur naturally)

Reply:
Let's add NOTE